packages/react-components/react-combobox/src/components/ComboButton/ComboButton.types.ts

```diff
@@ -36,4 +36,7 @@ export type ComboButtonProps = Partial<ComponentProps<ComboButtonSlots, 'content
 /**
  * State used in rendering ComboButton
  */
-export type ComboButtonState = ComponentState<ComboButtonSlots> & Pick<ComboboxState, 'appearance' | 'size'>;
+export type ComboButtonState = ComponentState<ComboButtonSlots> &
+  Pick<ComboboxState, 'appearance' | 'size'> & {
+    showingPlaceholder: boolean;
```

Comment:
not sure if we have styleguide on naming, but i think we do noun first right? `placeholderVisible` ?

Reply:
heh, I thought it was a bad name as I was writing it, but for some reason couldn't think of a better one 😅. `placeholderVisible` is much better 👍